utils/vpr_pbtype_arch_wrapper.py

```diff
@@ -221,7 +221,7 @@ def tile_xml(
     str
         The name of the top level pb_type (the tile).
     """
-    name, clocks, inputs, outputs = ports(pbtype_xml)
+    name, clocks, inputs, outputs, carry = ports(pbtype_xml)
```

Comment:
Given that carry doesn't seem used here, maybe this output argument oaught to be it's own function, rather than part of ports?

Reply:
@kgugala